lib/fetch.js

```diff
@@ -112,10 +112,21 @@ function csv(url, date) {
 async function headless(url, date) {
   // todo: figure out how to read from cache
   const browser = await puppeteer.launch();
-  const page = await browser.newPage();
-  await page.goto(url);
-
-  return { browser, page };
+ const pPage = await browser.newPage();
+ await pPage.setUserAgent('Mozilla/5.0 (Macintosh; Intel Mac OS X 10_13_2) AppleWebKit/537.36 (KHTML, like Gecko) Chrome/80.0.3987.132 Safari/537.36');
+ await pPage.setViewport({width: 1280, height: 800, isMobile: false})
+ const response = await pPage.goto(url, 
+  {
+      timeout: 25000, 
+      waitUntil: 'networkidle2'
+  });
+  
+  if (response._status < 400) {
+      await pPage.waitFor(3000);
+      let html = await pPage.content();
```

Comment:
This is going to need to be cached -- badly. Lets follow the same patterns for caching as used in `fetch`